allennlp/service/static/index.html

```diff
@@ -125,13 +125,21 @@
 
 class McOutput extends React.Component {
   render() {
-    const { rawOutput } = this.props;
+    const { passage, answer } = this.props;
+    const start = passage.indexOf(answer);
+    const head = passage.slice(0, start);
+    const tail = passage.slice(start + answer.length);
 
     return (
       <div className="model__content">
         <div className="form__field">
           <label>Answer</label>
-          <div className="model__content__summary">{ JSON.stringify(rawOutput, null, "  ") }</div>
+          <div className="model__content__summary">{ answer }</div>
+          <div>
+            <span>{head}</span>
+            <span><strong>{answer}</strong></span>
+            <span>{tail}</span>
```

Comment:
rather than doing it this way, I would add classes

```
<div className="passage">
  <span className="passage-head"> ...
  <span className="passage-answer"> ...
  <span className="passage-tail"> ...
</div>
```

and then use CSS to format them, that way it's much easier to play around with different formattings

Reply:
and also I think that's just better practice